src/lib.rs

```diff
@@ -1487,6 +1491,12 @@ impl Listener {
             panic!("ListenerStart failed, {:x}!\n", status);
         }
     }
+
+    pub fn close(&self) {
```

Comment:
Instead of this, I think you should just implement `Drop for Listener`.

Reply:
I've moved it to drop function.